### PR TITLE
SAK-31843 : Assignments / peer reviewer comments text field

### PR DESCRIPTION
**Pull request by frasese**
This PR does not increase the maximum allowed characters, but catches the exception when the current limit is exeeded.

Maybe it's better idea to change the colum REVIEW_COMMENT from VARCHAR(6000) to TEXT/CLOB?? But this will require a SQL migration script....

### CODE REVIEW

**Comment by ern** [comment]
This is exactly what will be done in the new Assignments Service I have been working on:
https://github.com/ern/sakai/blob/SAK-32101/assignment/assignment-api/api/src/java/org/sakaiproject/assignment/api/model/PeerAssessmentItem.java#L48

File: assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/AssignmentPeerAssessmentServiceImpl.java [hunk 1]
```diff
@@ -393,10 +394,14 @@ public PeerAssessmentAttachment getPeerAssessmentAttachment(final String submiss
...
 	}
 
-	public void savePeerAssessmentItem(PeerAssessmentItem item){
+	public void savePeerAssessmentItem(PeerAssessmentItem item) throws PeerReviewCommentTooLongException{
 		if(item != null && item.getAssessorUserId() != null && item.getSubmissionId() != null){
-			getHibernateTemplate().saveOrUpdate(item);
-			getHibernateTemplate().flush();
+			try {
+				getHibernateTemplate().saveOrUpdate(item);
+				getHibernateTemplate().flush();
```
Comment (ern) [inline assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/AssignmentPeerAssessmentServiceImpl.java]: This flush should not be needed here, as long as its in a transaction.

**Comment by ern** [comment]
@frasese I agree with you the better option here is to change the type, but given that this will happen in the new Assignments service not sure this is really needed. I am expecting to get the assignments changes in for sakai 12.

**Comment by frasese** [comment]
Ok, thanks. If we are going to change the type, my new Exception is no longer needed. 
So, I close this PR in favour of yours.

**Pull request closed without merging**